packages/mobile/src/paymentRequest/PaymentRequestConfirmation.tsx

```diff
@@ -113,11 +112,10 @@ class PaymentRequestConfirmation extends React.Component<Props> {
     const paymentInfo = {
       amount: amount.toString(),
       comment: this.state.comment || undefined,
-      timestamp: new Date(),
```

Comment:
I now realize the `timestamp` never ended up being stored in Firebase RTDB, because it doesn't support it directly.

In that case I propose we do the following:
1. drop the `timestamp` field
2. add a `createdAt` field, letting the Firebase set the value on the server side using `firebase.database.ServerValue.TIMESTAMP`
3. possibly adding an `updatedAt` field, we could use a cloud function for this

What do you think?

Reply:
I didn't know about `firebase.database.ServerValue.TIMESTAMP`, looks cool.  I'm not sure of how much value this `timestamp/createdAt` field brings though (it hasn't been working since forever and noone noticed), but I can't see why not either.

I didn't add the `updatedAt` field for now since it doesn't sound too high priority, but it wouldn't be too hard to add from the cloud function that sends the notifications I guess.